recommendation-generator/src/main/java/edu/kit/kastel/mcse/ardoco/core/recommendationgenerator/extractors/CompoundRecommendationExtractor.java

```diff
@@ -42,38 +42,38 @@ public void run() {
             var modelState = modelStatesData.getModelState(model);
             var recommendationState = recommendationStates.getRecommendationState(modelState.getMetamodel());
 
-            createRecommendationInstancesFromPhraseNounMappings(textState, recommendationState, modelState);
-            findMorePhrasesForRecommendationInstances(textState, recommendationState, modelState);
+            createRecommendationInstancesFromCompoundNounMappings(textState, recommendationState, modelState);
+            findMoreCompoundsForRecommendationInstances(textState, recommendationState, modelState);
             findSpecialNamedEntitities(textState, recommendationState);
         }
     }
 
     /**
-     * Look at NounMappings and add RecommendedInstances, if a NounMapping was created because of a phrase (in
+     * Look at NounMappings and add RecommendedInstances, if a NounMapping was created because of a compound (in
      * text-extraction)
      */
-    private void createRecommendationInstancesFromPhraseNounMappings(TextState textState, RecommendationState recommendationState,
+    private void createRecommendationInstancesFromCompoundNounMappings(TextState textState, RecommendationState recommendationState,
             ModelExtractionState modelState) {
         for (var nounMapping : textState.getNounMappings()) {
-            if (nounMapping.isPhrase()) {
+            if (nounMapping.isCompound()) {
```

Comment:
Just seen here. can a nounmapping be a "compound" or are the compounds the "words"